hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServices.java

```diff
@@ -1099,4 +1099,63 @@ private RMWebServices prepareWebServiceForValidation(
     return  webService;
   }
 
+  @Test
+  public void testClusterSchedulerOverviewFifo() throws JSONException, Exception {
+    WebResource r = resource();
+    ClientResponse response = r.path("ws").path("v1").path("cluster")
+        .path("scheduler-overview").accept(MediaType.APPLICATION_JSON)
+        .get(ClientResponse.class);
+
+    assertEquals(MediaType.APPLICATION_JSON + "; " + JettyUtils.UTF_8,
+        response.getType().toString());
+    JSONObject json = response.getEntity(JSONObject.class);
+    verifyClusterSchedulerFifoOverView(json);
+    System.out.println(json);
```

Comment:
Remove debugging

Reply:
Thank you very much for your help reviewing the code, I will fix it.

Reply:
When I get the sub-cluster scheduling information, I define a new interface `RMWebServices#getSchedulerOverview`
I submitted a separate PR for the code of this part of the new interface(https://github.com/apache/hadoop/pull/4947), because modifying the Router About Web Page code seems to be different from the scope of the new interface, can you help to review (https://github.com/apache/hadoop/pull/4947), Thank you very much!